pytab/core.py

```diff
@@ -300,6 +307,32 @@ def cli(
     system_info = hwi.get_system_info()
     click.echo(" success!")
 
+    # Logic for GPU Selection
+    gpus = system_info["gpu"]["gpu_elements"]
+    if len(gpus) > 1 and "gpu_id" not in globals():
```

Comment:
Note that `gpu_id` is not a global Variable. Instead it should be available as a normal Variable.
Doing
```python
if ... and not gpu_id:
```
 should probably do the Job.